Documentation/subcommands/run.md

```diff
@@ -84,16 +84,79 @@ rkt: warning: signature verification has been disabled
 
 ## Mount Volumes into a Pod
 
-Volumes are defined in each ACI and are referenced by name. Volumes can be exposed from the host into the pod (`host`) or initialized as empty storage to be accessed locally within the pod (`empty` pending [rkt #378][rkt #378]). Each volume can be selectively mounted into each application at differing mount points or not mounted into specific apps at all.
+Each ACI can define a [list of mount points](https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema) that the app is expecting external data to be mounted into:
 
-[rkt #378]: https://github.com/coreos/rkt/issues/378
+```json
+{
+    "acKind": "ImageManifest",
+    "name": "example.com/app1",
+    ...
+    "app": {
+        ...
+        "mountPoints": [
+            {
+                "name": "data",
+                "path": "/var/data",
+                "readOnly": false
+            }
+        ]
+    }
+    ...
+}
+```
+
+To fulfill these mount points, volumes are used. There are today two kinds of volumes:
```

Comment:
Maybe mention that volume and mount points have names and that is how they are matched. Or is this a part of a spec and we assume that user knows it?